src/Analysim.Web/ClientApp/src/app/template/password-reset/password-reset.component.html

```diff
@@ -0,0 +1,39 @@
+<div class="email-template-wrapper">
```

Comment:
@Ahkeelah did you create a new component for password reset?

Reply:
No, that was a mock up for what it would look like. 
We are not going to be using this, it's just to show Dr.Anca what it would look like.

Reply:
Ok! Let's remember to delete it once it's done :)